jenkins-x.yml

```diff
@@ -111,6 +111,15 @@ pipelineConfig:
                     - --cache-repo=gcr.io/jenkinsxio/cache
                     - --cache=true
                     - --cache-dir=/workspace
+                - name: build-and-push-nodejs12x
```

Comment:
This is failing due to the new dockerfile not being on this branch. =)

Reply:
gah...i really need to remember to reset my master before branching off again 🤦‍♂